models/Tool/CustomReport/Config.php

```diff
@@ -70,62 +70,62 @@ class Config extends Model\AbstractModel implements \JsonSerializable
     /**
      * @var bool
      */
-    protected $menuShortcut;
+    protected $menuShortcut = true;
 
     /**
      * @var string
      */
-    protected $reportClass;
+    protected $reportClass = '';
 
     /**
      * @var string
      */
-    protected $chartType;
+    protected $chartType = '';
 
     /**
      * @var string
      */
-    protected $pieColumn;
+    protected $pieColumn = '';
 
     /**
      * @var string
      */
-    protected $pieLabelColumn;
+    protected $pieLabelColumn = '';
 
     /**
      * @var string
      */
-    protected $xAxis;
+    protected $xAxis = '';
 
     /**
      * @var string|array
      */
-    protected $yAxis;
+    protected $yAxis = [];
 
     /**
-     * @var int
+     * @var int|null
      */
     protected $modificationDate;
 
     /**
-     * @var int
+     * @var int|null
      */
     protected $creationDate;
 
     /**
      * @var bool
      */
-    protected $shareGlobally;
+    protected $shareGlobally = true;
```

Comment:
```suggestion
    protected $shareGlobally = false;
```
I'd set this `false` to not change the behavior.

Reply:
If you create a CustomReport in the backend the checkbox is checked by default. So I think it would be good to have the same behavior in PHP. But I can change it.

Reply:
@blankse ah, sorry, I thought it's off by default - thanks for clarifying, nothing else to change then 😉 Thanks!

Reply:
Oh, is not checked... I change it. Hmm I tought I checked this...

Reply:
👍 just wanted to double check as well 😊 Thanks!

Reply:
For me it's enabled by default 🤔 So seems ok to me.

Reply:
Okay, in 6.9 it was unchecked. In 10.2 it is checked by default. :)